Cargo.toml

```diff
@@ -11,6 +11,7 @@ arc-swap = ">=0.4.4"
 clap = { version = "2.33.1", features=["wrap_help"] }
 epoll = ">=4.0.1"
 futures = { version = "0.3.5", features = ["thread-pool"] }
+hypervisor = { path = "hypervisor" }
```

Comment:
Could you please add an extra commit introducing the new `hypervisor` crate without anything inside?
This commit should be the first of the entire patchset, and it will be the opportunity to describe the goal of this new crate in the commit message.

Reply:
@sameo I guess I can add one more extra commit just to add the skeleton of the hypervisor crate. vcpu would be the next commit.